src/app/products/loan-products/loan-product-stepper/loan-product-payment-strategy-step/payment-allocation-model.ts

```diff
@@ -41,6 +41,12 @@ export class AdvancedPaymentAllocation {
   futureInstallmentAllocationRules: FutureInstallmentAllocationRule[];
 }
 
+export class AdvancePaymentAllocationData {
+  transactionTypes: PaymentAllocationTransactionType[];
```

Comment:
It would be better to move these changes into a separate PR. They are not really related to tooltips